osu.Game/Utils/HumanizerUtils.cs

```diff
@@ -26,5 +26,32 @@ public static string Humanize(DateTimeOffset input)
                 return input.Humanize(culture: new CultureInfo("en-US"));
             }
         }
+
+        /// <summary>
+        /// Turns the current or provided big number into a readable string
+        /// </summary>
+        /// <param name="input">The number to be humanized</param>
+        /// <returns>simplified number with a suffix</returns>
+        public static string Humanize(long input)
+        {
+            const int k = 1000;
+
+            var suffixes = new[]
```

Comment:
```suggestion
            const string suffixes = new[]
```

Reply:
It doesn't work like this

Reply:
at very least you want to move this outside of the method. the way you're doing it, this needs to be reallocated every call.

`static readonly suffixes = new[] ...`